osde2e/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
 
+# THIS FILE IS GENERATED BY BOILERPLATE. DO NOT EDIT.
+FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+    
 ENV PKG=/go/src/github.com/openshift/aws-vpce-operator/
-WORKDIR ${PKG}
-
-# compile test binary
+WORKDIR ${PKG}    
 COPY . .
-
+    
 FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
This needs to be a specific image tag, not latest.

Reply:
[We had a ](https://issues.redhat.com/browse/SDCICD-926)